baseplate/integration/pyramid.py

```diff
@@ -62,6 +62,25 @@ def baseplate_tween(request):
     return baseplate_tween
 
 
+class BaseplateEvent(object):
+
+    def __init__(self, request):
+        self.request = request
+
+
+class ServerSpanInitialized(BaseplateEvent):
+    """Event that Baseplate fires after creating the ServerSpan for a Request.
+
+    This event will be emitted before the Request is passed along to it's
```

Comment:
its

Reply:
Doh, I should have caught that:

https://youtu.be/yc2udEpyPpU?t=20